benchmarks/build.gradle

```diff
@@ -9,11 +10,24 @@ run.enabled = false
 buildscript {
     repositories {
         mavenCentral()
+        maven {
+            url "https://plugins.gradle.org/m2/"
+        }
     }
     dependencies {
         classpath libraries.protobuf_plugin
+        classpath "me.champeau.gradle:jmh-gradle-plugin:0.2.0"
```

Comment:
Why aren't we using the new plugin syntax as the plugin suggests?

plugins {
   id 'me.champeau.gradle.jmh' version '0.2.0'
}